src/inverseRobot.js

```diff
@@ -8,6 +8,16 @@
 
 function inverseRobot(robot) {
   // write code here
-}
+  const inverse = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(inverse, robot[key])) {
+      return null;
+    }
 
+    inverse[robot[key]] = key;
+  }
+
+  return inverse;
```

Comment:
You've correctly returned the new object. However, consider adding a brief comment explaining what this return statement does.